docs/src/theme/NotFound/Content/index.js

```diff
@@ -36,50 +36,45 @@ I found the broken link on https://...
       <div className="row">
         <div className="col col--6 col--offset-3">
           <Heading as="h1" className="hero__title">
-            <Translate
-              id="theme.NotFound.title"
-              description="The title of the 404 page"
-            >
-              Sorry, looks like this page was lost at sea!
-            </Translate>
+            Sorry, this link is broken.
           </Heading>
           <p>
-            <Translate
-              id="theme.NotFound.p1"
-              description="The first paragraph of the 404 page"
-            >
-              We could not find what you were looking for.
-            </Translate>
-          </p>
-          <p>
-            <Translate
-              id="theme.NotFound.p2"
-              description="The 2nd paragraph of the 404 page"
-            >
-              This is a bug, please help us fix it!
-            </Translate>
+            <strong>Will you help us fix it?</strong>{" "}
+            All you need to do is open an issue with some information about where you just clicked this link.
           </p>
-          <p>
-            <Translate
-              id="theme.NotFound.p3"
-              description="The 3rd paragraph of the 404 page"
-            >
-              When creating the issue, please make sure to reference the following path:
-            </Translate>
-          </p>
-          <pre className="language-text">
-            {currentPath}
-          </pre>
-
-          <div className="text--center">
-            <a
-              href={githubIssueUrl}
-              target="_blank"
-              rel="noopener noreferrer"
-              className="button button--primary button--sm"
-            >
-              Report a Bug
-            </a>
+          <Heading as="h3">
+            Step 1: Get the URL you just came from
+          </Heading>
+          <div>
+            <p>
+              This is the most important step. We don't track you and so we don't know which site you just came from.
```

Comment:
Should we qualify that we're not interested in *any* site, and only those on our domains, like `openpolicyagent.org`. Or is that insulting the user's intelligence? 😅

Reply:
If someone finds a broken link on a 3rd party site, that's the most interesting for us since we have our own internal broken link scanning. So we're actually interested in the source here, and then we can decide if we want to redirect, or PR/request to get the source updated.